build.sbt

```diff
@@ -5,9 +5,6 @@ ThisBuild / organization      := "it.pagopa"
 ThisBuild / organizationName  := "Pagopa S.p.A."
 ThisBuild / version           := ComputeVersion.version
 Global / onChangedBuildSource := ReloadOnSourceChanges
-ThisBuild / githubOwner       := "pagopa"
-ThisBuild / githubRepository  := "interop-commons"
-ThisBuild / resolvers += Resolver.githubPackages("pagopa")
```

Comment:
???

Reply:
This PR is merging in 1.0.x, are you sure the destination branch is the correct one?